citesphere/src/main/java/edu/asu/diging/citesphere/core/service/giles/impl/GilesUploadCheckerImpl.java

```diff
@@ -153,6 +168,7 @@ public void checkUploads() {
                         logger.error("Could not deserialize response.", e);
                         upload.setDocumentStatus(GilesStatus.FAILED);
                         checkedUploads.add(upload);
+                        continue;
```

Comment:
if this is added here, then processed does not need to be initialized in line 164, does it?